.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,84 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+name: Arquillian Integration Tests
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
This will add 6 jobs to each PR. I'm not sure if we want to do that, but welcome feedback.

Reply:
What if we just run SE 21 on the integration tests on each platform automatically and have a complete integration-test job that can be run manually that covers the latest LTS and the two previous? The full set is running in 6.5 minutes which is not too bad.